right-click-for-superpowers/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0" />
+    <meta name="author" content="Jason Mayes" />
+    <title>Gemma 2B LLM Right Click Superpowers</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
```

Comment:
Please use relative urls if possible, to enable the demo to run on chrome.dev (they run on a deep url like chrome.dev/my-demo)

Reply:
This is a relative URL. Would work from whatever folder it is served from so long as the 3 files are in the same folder.